### ISSUE TITLE
Can AlexNet be applied to the CIFAR-10 dataset by using the code you published? Thank you!

### ISSUE THREAD

**Issue by realcly** [yjlee22/FedShare#3]
I have learned a lot after reading your paper and code. Thank you for sharing. I noticed that there was AlexNet network in the net.py file, but after I changed the code a little bit to use AlexNet, I had some problems, such as the test accuracy was not as good as CNN or even very low, etc. Can AlexNet be applied to the CIFAR-10 dataset by using the code you published? Thank you!

**Comment by yjlee22** [comment]
Unfortunately, this is not my paper. I just implemented with pytorch to reproduce the result of the paper. I recommend that you do the following:
Before:
from src.nets import MLP, CNN_v1, CNN_v2
net_glob = CNN_v2(args=args).to(args.device)
After
import torch
from torchvision.models import alexnet
net_glob = alexnet(num_classes=args.num_classes).to(args.device)
if args.dataset == "mnist":
    net_glob.Conv2d = torch.nn.Conv2d(1, 64, kernel_size=11, stride=4, padding=2)
